crates/uv-extract/src/stream.rs

```diff
@@ -104,11 +106,99 @@ pub async fn untar<R: tokio::io::AsyncBufRead + Unpin>(
     reader: R,
     target: impl AsRef<Path>,
 ) -> Result<(), Error> {
+    /// Determine the path at which the given tar entry will be unpacked, when unpacking into `dst`.
+    ///
+    /// This is effectively a vendored version of some logic in `tokio_tar`, within `unpack_in`.
```

Comment:
Do we need a license?

Reply:
I think it's sufficient to link to the source here.